addons/website_forum/static/src/js/website_forum.js

```diff
@@ -303,6 +305,21 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
     _onTagFollowBoxMouseLeave: function (ev) {
         $(ev.currentTarget).find('.o_forum_tag_follow_box').stop().fadeOut().css('display', 'none');
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onClickTagFollow: function (ev) {
+        const closestBtn = ev.target.closest("button");
+        if (
+            closestBtn &&
+            (closestBtn.classList.contains("js_follow_btn") ||
+                closestBtn.classList.contains("js_unfollow_btn"))
+        ) {
+            ev.currentTarget.querySelector(".o_js_forum_tag_link").classList.toggle("text-dark");
+            ev.currentTarget.querySelector(".o_js_forum_tag_link").classList.toggle("fw-bold");
```

Comment:
it would be better so use a class for the state and to have the style in CSS
(that way we separate the style from the logic)

but I'm picky so it's ok if you prefer like that